SpotifyAPI/Local/Models/SpotifyUri.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpotifyAPI.Local.Models
+{
+    public class SpotifyUri
+    {
+        public string Base { get; internal set; }
+        public string Type { get; internal set; }
+        public string Id { get; internal set; }
+
+        public SpotifyUri(string Uri)
+        {
+            string[] props = Uri.Split(':');
```

Comment:
One thing we should propably do: Error-Handling

If `props.length` is not 3, throw a `InvalidOperationException` or sth similar (is there a better exception out there?)